tasklist/app/views/tasks/index.html.erb

```diff
@@ -7,6 +5,10 @@
       <% @tasks.each do |task| %>
         <li>
           <%= link_to task.name, "/tasks/#{task.id}" %> ✍ <br>
+            <% if !task.person.nil? %>
+            <%= link_to task.person.name, "/people/#{task.person_id}/tasks"  %> ℹ︎
```

Comment:
You still want to indent code that is inside of a conditional block even when it is in your view